library/src/main/java/com/bumptech/glide/load/resource/gif/GifFrameLoader.java

```diff
@@ -103,6 +104,7 @@ Bitmap getFirstFrame() {
   }
 
   void subscribe(FrameCallback frameCallback) {
+    unsubscribeRegisteredCallback();
```

Comment:
What if subscribe is never called again?

Reply:
This is just fail safe.
Normally, unsubscribeRegisteredCallback() is called inside onFrameReady() and then callbacks will be unsubscribed,.
When an user click image repeat quickly, subscribe() may be called before onFrameReady() is called..